test/evergreen.yml

```diff
@@ -1614,6 +1614,24 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: time-shift-sensitivity-test
+    depends_on:
+      - name: compile
+        vars:
+          posix_configure_flags: --enable-strict
+    commands:
+      - func: "fetch artifacts"
+        vars:
+          posix_configure_flags: --enable-strict
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            ../test/csuite/time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1
```

Comment:
Assume the library file exists on the `ubuntu1804-test` distro?

Reply:
faketime library has been installed in ubuntu18.04 and rhel80 as per ticket https://jira.mongodb.org/browse/BUILD-9213